Tasks/distinct.js

```diff
@@ -1,19 +1,16 @@
 // Return an array without duplicates
 
-DISTINCT = (data) => {
-  console.log({ data });
-  A = new Set();
-  w = 0;
-  data.forEach((a) => {
-    if (A.has(a)) {
-      delete data[w];
-    } else {
-      A.add(a);
-    }
-    w++;
-  });
-  return data.filter
-  (x => typeof x === 'number');
+"use strict";
+
+const distinct = (array) => {
+  console.log({ array });
```

Comment:
I think the logging here is only needed for the face-to-face view of the collection and is not needed in the code itself
```suggestion
```